src/theme/shadow/specs/default-shadow-set.ts

```diff
@@ -0,0 +1,30 @@
+import { getPrimitiveColour } from "../../colour-primitive/theme-helper";
+import { getSemanticColour } from "../../colour-semantic/theme-helper";
+import { StyledComponentProps } from "../../helpers";
+import { PrimitiveColourSet, SemanticColourSet } from "../../types";
+import { ShadowSet } from "../types";
+
+const createShadow =
+    (shadowDef: string, colourKey: string, alpha: number) =>
```

Comment:
we can type colourKey here to be from `PrimitiveColourSet` or `SemanticColourSet` for type safety